src/main/resources/io/jenkins/plugins/folderauth/FolderAuthorizationStrategyManagementLink/index.jelly

```diff
@@ -22,21 +22,32 @@
                     <input type="text" placeholder="${%filterPlaceholder}" class="filter" id="globalRoleFilter"/>
                     <div class="role-container" id="globalRoleContainer">
                         <j:forEach items="${it.globalRoles}" var="globalRole" indexVar="index">
-                            <div class="role" roleName="${globalRole.name}">
+                            <div class="role" roleName="${globalRole.name}" id="global-role-${globalRole.name}">
                                 ${%name}: ${globalRole.name}
                                 <br/>
                                 ${%sids}: ${globalRole.getSidsCommaSeparated()}
                                 <form action="${rootURL}/${it.urlName}/assignSidToGlobalRole" method="POST">
                                     <div class="form-row">
-                                        <label for="assign-sid-global-${index}" style="margin-right: 10px;">
+                                        <label for="assign-sid-global-${globalRole.name}" style="margin-right: 10px;">
                                             ${%assignSid}
                                         </label>
-                                        <input id="assign-sid-global-${index}" type="text" name="sid"/>
+                                        <input id="assign-sid-global-input-${globalRole.name}" type="text" name="sid"/>
                                         <input type="hidden" value="${globalRole.name}" name="roleName"/>
-                                        <input type="submit" style="margin-left: 10px;" value="${%submit}"
+                                        <input id="assign-sid-global-submit-${globalRole.name}" type="submit" style="margin-left: 10px;" value="${%submit}"
                                                class="submit"/>
                                     </div>
                                 </form>
+                                <form action="${rootUrl}/${it.urlName}/removeSidFromGlobalRole" method="POST">
+                                    <div class="form-row">
```

Comment:
ok i've since changed it to be of the format you pasted above. here's a screenshot
![image](https://user-images.githubusercontent.com/6691227/68383529-c7f71a80-0190-11ea-9b36-3d12a67e88df.png)
also, moving forward it'll be good if we can revamp the whole UI. I would like the ability to modify permissions for added roles and the current UI doesn't allow for that.